notify_send.py

```diff
@@ -572,11 +577,12 @@ def prepare_notification(buffer, nick, message):
     message = escape_slashes(message)
 
     icon = weechat.config_get_plugin('icon')
+    desktop_entry = weechat.config_get_plugin('icon')
```

Comment:
Shouldn't this be
```python
desktop_entry = weechat.config_get_plugin('desktop_entry')
```
instead of
```python
desktop_entry = weechat.config_get_plugin('icon')
```
?

If so, how come the current unit tests pass and do not fail because of this?